minard/views.py

```diff
@@ -104,6 +104,10 @@ def state(run=None):
     if run_state['caen'] is not None:
         caen_state = detector_state.get_caen_state(run_state['caen'])
 
+    #tubii_state = None
+    #if run_state['tubii'] is not None:
```

Comment:
Issue seems to go away if use the detector db rather than the test db so I've updated the PR.
I'm still going to get Eric to check it works for him before it should be merged though.